### PR TITLE
Feature/change var

### PR DESCRIPTION
**Pull request by piquev**
We should update most of var in let.

I changed few var to const because i though it was more appropriate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,8 +102,8 @@ for (var i = 0; i < 525600; i++) {
 **Good**:
 ```javascript
 // Declare them as capitalized `var` globals.
```
Comment (vsemozhetbyt) [inline README.md]: Maybe '// Declare them as capitalized `const` globals.' as well.

Comment (piquev) [inline README.md]: Yep you are right

File: README.md [hunk 2]
```diff
@@ -101,9 +101,9 @@ for (var i = 0; i < 525600; i++) {
 
 **Good**:
 ```javascript
-// Declare them as capitalized `var` globals.
-var MINUTES_IN_A_YEAR = 525600;
-for (var i = 0; i < MINUTES_IN_A_YEAR; i++) {
+// Declare them as capitalized `const` globals.
+const MINUTES_IN_A_YEAR = 525600;
```
Comment (patcon) [inline README.md]: came into the issues wondering about the same thing ;)

**Comment by ryanmcdermott** [comment]
Thanks so much at looking into this Valère! I appreciate your good insights. Keep up the PRs!

**Pull request closed without merging**